Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny4/git-issue46.dfy.expect

```diff
@@ -1,2 +1,2 @@
 
-Dafny program verifier finished with 1 verified, 0 errors
+Dafny program verifier finished with 0 verified, 0 errors
```

Comment:
I'm surprised here because the test is only about resolution. How come there was something verified before?